chingsley/ireporter
# ch(setupDB): connect end points to the db

#### What does this PR do?
This pull request deals with the connection of the DB to persist data

#### Description of Task to be completed?
- install postgress 
- run SQL commands in the terminal to create a database
- set up and run a schema.js file to create the tables
- connect the endpoints to the database 

#### How should this be manually tested?
- clone the repo using git clone
- cd into the root folder of the project and run npm install to install the dependencies
- run npm test to see the tests passing
- to test the endpoints, run npm start, then use postman to test the endpoints

#### What are the relevant pivotal tracker stories?
#162573557, #162586217

#### Screenshots (if appropriate)

#### Questions:
